Marlin/src/pins/stm32f4/pins_BTT_SKR_PRO_common.h

```diff
@@ -98,6 +98,13 @@
   #define Z_MIN_PROBE_PIN                   PA2
 #endif
 
+//
+// Filament Runout Sensor
+//
+#ifndef FIL_RUNOUT_PIN
+  #define FIL_RUNOUT_PIN                    PE15
+#endif
+
```

Comment:
As long as this is being added, you should add all three.

```suggestion
#ifndef FIL_RUNOUT2_PIN
  #define FIL_RUNOUT2_PIN                    PE10
#endif
#ifndef FIL_RUNOUT3_PIN
  #define FIL_RUNOUT3_PIN                    PG5
#endif

```